# Generated for NaoQi method ALAudioPlayer.playFileInLoop
# Plays a wav or mp3 file in loop. Asynchronous.

# Path of the sound file
string fileName
---
# A string that summarises the usage of the method.
int32 value


